static/src/javascripts/projects/commercial/modules/third-party-tags/outbrain-sections.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+const sections = ['politics', 'world', 'business', 'commentisfree'];
+
+const getSection = function(section: string): string {
+    const sectionLower = section.toLowerCase();
+    return /news/.test(sectionLower) || sections.indexOf(sectionLower) !== -1
```

Comment:
Not mandatory, but could be written better as

```js
return /news/.test(sectionLower) || sections.includes(sectionLower)
```

now. Leave it up to you whether you'd like to make this change or not.